docs/SECURITY-PROCESS.md

```diff
@@ -269,3 +269,10 @@ timeout value or otherwise) are not considered security problems. Applications
 are supposed to already handle situations when the transfer loop legitimately
 consumes 100% CPU time, so while a prolonged such busy-loop is a nasty bug, we
 do not consider it a security problem.
+
+## Saving files
+
+A user of curl or libcurl on multi-user systems **should not** keep critical
+files in directories in which you allow curl or libcurl to save or update
+files - if other users also have access to the same directory. curl cannot
+completely protect such files against attacks.
```

Comment:
In my opinion this is still unclear and I'm not sure that what I suggested earlier is any better. I think a user who reads this may not be able to comprehend the full scope. For example (if I understand correctly) regardless of whether there's critical files in the directory another user with access to the directory could still attack. Maybe instead of getting more into the weeds it could be shortened to one sentence concise like "curl cannot protect against attacks where an attacker has write access to the directory where curl is directed to save files." ?